src/components/action-bar/action-bar-item/action-bar-overflow-menu.tsx

```
@@ -28,6 +28,9 @@ export class ActionBarOverflowMenu {
     @Prop({ reflect: true })
     public openDirection: OpenDirection = 'bottom-end';
 
+    @Prop()
+    public actionBarShrinked: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)